src/FormV2/UserCreation/CreateBeneficiaryType.php

```diff
@@ -49,7 +49,7 @@ public function addIdentityFields(FormBuilderInterface $builder, ?\DateTime $bir
             ->add('dateNaissance', BirthdayType::class, [
                 'label' => 'birthDate',
                 'row_attr' => ['class' => 'mt-3'],
-                'data' => $birthDate ?? new \DateTime('01/01/1975'),
+                'data' => $birthDate ?? Beneficiaire::getDefaultBirthDate(),
```

Comment:
Maybee this should live inside benefconstructor instead of in the form type no ?